src/components/switch/_switch.scss

```diff
@@ -0,0 +1,106 @@
+/* ===============Switch================== */
```

Comment:
Is this theme reviewed by XD?

Reply:
Yes, comments given by XD was incorporated in the theme